projects/gov-ui/src/lib/components/gov-uk-checkbox/gov-uk-checkbox.component.ts

```diff
@@ -21,7 +21,7 @@ import {FormGroup} from '@angular/forms';
 export class GovUkCheckboxComponent implements OnInit {
   constructor() { }
   @Input() group: FormGroup;
-  @Input() config: {value: string, label: string, hint: string; name: string; focusOn: string; id: string; classes: string};
+  @Input() config: {value: string, label: string, hint: string; name: string; focusOn: string; id: string; classes: string; hiddenLabelContext: string};
```

Comment:
This is still 151 characters long. I'm surprised you're not getting lint errors because you'd be getting them in other projects. I guess we have yet another inconsistency with the `tslint` files.

Would you mind doing a spot of wrapping here, please, @RiteshHMCTS, to avoid this being a problem when the `tslint` files get standardised across the projects?

Reply:
split the line